doc/design/architecture.md

```diff
@@ -120,6 +127,14 @@ None --> UpgradeAvailable --> UpgradePending --> AtLatestKnown -+
 | UpgradePending   | `InstallPlan` has been created (referenced in `status.installplan`) to install a new CSV                   |
 | AtLatestKnown    | `status.installedCSV` matches the latest available CSV in catalog                                             |
 
+## Package Server
+
+The Package Server provides a veneer API over the CatalogSource resources and the information they provide to ease user interaction. 
+
+It is not neccessary to run this for OLM to function, but enhances the user experience by providing the information needed for Subscription generation to the user. In a Production scenario where no discovery work is needed and all Subscriptions are predefined, this may be removed.
+
+OpenShift/OKD GUIs will use this API extention to present operator package information such as CSVs, packages and CRDs found in the CatalogSource to users. Additionally, `kubectl` and `oc` users are able to issue commands like `kubectl get PackageManifests` to list out available Operators and then run `kubectl describe` on those resources to find the appropriate data to build a Subscription to it.
```

Comment:
This is upstream doc so I don't think we should reference to a specific platform like OpenShift/OKD here. I would prefer to remove all OpenShift/OKD and `oc` from this doc.

Reply:
Good point, I rewrote that paragraph.